client/src/modules/patients/registry/registry.js

```diff
@@ -202,23 +196,35 @@ function PatientRegistryController($state, Patients, Notify, AppCache,
     Receipts.patient(uuid);
   }
 
-  // startup function. Checks for cached filters and loads them.  This behavior could be changed.
-  function startup() {
+  // format Export Parameters
+  function formatExportParameters(type) {
+    return { renderer: type || 'pdf', lang: Languages.key };
+  }
+
+  // display the patient registry printable report
+  vm.openJournalReport = function openJournalReport() {
```

Comment:
I don't think this should be called `openJournalReport` since it isn't the journal.  Maybe `PatientReport`?